src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.FruitTransfer;
+import core.basesyntax.Operation;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransfer> convertToTransfer(List<String> inputReport) {
+        List<FruitTransfer> transferList = new ArrayList<>();
+        List<String[]> collectedLinesInfo = inputReport.stream()
+                .map(s -> s.split(","))
+                .skip(1)
+                .toList();
+
+        for (String[] line : collectedLinesInfo) {
+            Operation operations = Operation.getOperations(line[0]);
+            String fruit = line[1];
+            int quantity = Integer.parseInt(line[2]);
+            transferList.add(new FruitTransfer(operations, fruit, quantity));
```

Comment:
Move indexes to the constants
```suggestion
            Operation operation = Operation.getOperations(line[0]);
            String fruit = line[1];
            int quantity = Integer.parseInt(line[2]);
            transferList.add(new FruitTransfer(operation, fruit, quantity));
```